boards/FSM/Src/Io/Io_MSP3002K5P3N1.c

```diff
@@ -1,6 +1,6 @@
-#include <stdbool.h>
 #include "Io_Adc.h"
 #include "main.h"
+#include "Io_MSP3002K5P3N1.h"
 
 bool Io_MSP3002K5P3N1_IsBrakeActuated(void)
```

Comment:
Hmm, it seems like this function is duplicated between here and `Io_Brake`? Could we remove one of them?

Reply:
Also I'm not entirely clear on what "actuated" means here?

Reply:
Would using IsBrakeApplied be clearer?

Reply:
The rule book uses actuated. I'm fine with either.

![actuated](https://user-images.githubusercontent.com/32230980/84611292-3e1fd600-ae72-11ea-952c-00e41de817a9.png)